src/mmw/js/src/modeling/models.js

```diff
@@ -652,6 +656,9 @@ var ScenarioModel = Backbone.Model.extend({
 
                 pollSuccess: function() {
                     self.setResults();
+                    if (self.waitToRefresh) {
+                        self.get('results').trigger('reset');
```

Comment:
Would `self.get('results').reset()` work here? Also, if there a case in `pollSuccess` where we don't want to trigger a reset? In other words, do we need the conditional or can we just call `reset` always?